doc/nrf/ug_bt_mesh_configuring.rst

```diff
@@ -34,3 +34,10 @@ Mesh models
 
 The |NCS| Bluetooth mesh model implementations are optional features, and each model has individual Kconfig options that must be explicitly enabled.
 See :ref:`bt_mesh_models` for details.
+
+Additional configuration options
+================================
+
+This section lists additional configuration options that can be used to configure behavior and performance of Bluetooth mesh. This section will be expanded in future to include relavent configuration options.
+
+* :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` - Enables partial erase on supported hardware platforms. Partial erase allows flash page erase operation to be split into several small chunks preventing longer CPU stalls. This helps in improving responsiveness of mesh node during defragmentation of storage areas used by settings subsystem.
```

Comment:
Should we move CONFIG_BT_SETTINGS into this section too? (From line 29-30)

Reply:
No, CONFIG_BT_SETTINGS is mandatory because of persistence storage requirements for RPL and model states. So, in that sense, it is a must-have default. But we still need the optionality of CONFIG_BT_SETTINGS for testing purposes. So, I feel it is good as is.